release.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Bash version should >= 4 to be able to run this script.
 
-IMAGE="budtmo/docker-android"
+IMAGE="${DOCKER_ORG:-budtmo}/docker-android"
```

Comment:
Should allow forks to automate their builds :)

Reply:
If I understand correctly, the user can push on their own docker hub? should `:-budtmo` need to be deleted?

Reply:
I keep that default value to not break your Travis-ci build.
Personally, I think we should respect the value `budtmo` :D

Reply:
but it wont work, will it? because e.g. my DOCKER_ORG is budtmo and yours is trinhpham not trinhpham-budtmo. or do I miss something?

Reply:
It a trick of bash syntax to make a default variable (ref: https://unix.stackexchange.com/a/122848), so you'll have `budtmo` as default value.

Reply:
@budtmo do you need any more explanation?

Reply:
Hi, sorry for late response, I will check your PR and merge it asap. Thank you again for the good work!